File: Client_Root/Client/Assets/Scripts/Network/Messages/ToLobby/SelectNormalGameToL.cs
using FlatBuffers;

public class SelectNormalGameToL : IMessage
{
    public const ushort MESSAGE_ID = MessageID.SelectNormalGameToL_ID;

    public override ushort GetID()
    {
        return MESSAGE_ID;
    }

    public override IMessage Clone()
    {
        return null; 
    }

    public override byte[] Serialize()
    {
        FBS.SelectNormalGameToL.StartSelectNormalGameToL(m_Builder);
        var data = FBS.SelectNormalGameToL.EndSelectNormalGameToL(m_Builder);

        m_Builder.Finish(data.Value);

        return m_Builder.SizedByteArray();
    }

    public override bool Deserialize(byte[] bytes)
    {
        var buf = new ByteBuffer(bytes);

        var data = FBS.SelectNormalGameToL.GetRootAsSelectNormalGameToL(buf);

        return true;
    }
}

File: Server_Root/Server/Server/Messages/ToLobby/SelectNormalGameToL.cpp
#include "stdafx.h"
#include "SelectNormalGameToL.h"
#include "../../../FBSFiles/SelectNormalGameToL_generated.h"

SelectNormalGameToL::SelectNormalGameToL()
{
}

SelectNormalGameToL::~SelectNormalGameToL()
{
}

unsigned short SelectNormalGameToL::GetID()
{
	return MESSAGE_ID;
}

IMessage* SelectNormalGameToL::Clone()
{
	return NULL;
}

const char* SelectNormalGameToL::Serialize(int* pLength)
{
	FBS::SelectNormalGameToLBuilder data_builder(m_Builder);
	auto data = data_builder.Finish();

	m_Builder.Finish(data);

	*pLength = m_Builder.GetSize();

	return (char*)m_Builder.GetBufferPointer();
}

bool SelectNormalGameToL::Deserialize(const char* pChar)
{
	auto data = flatbuffers::GetRoot<FBS::SelectNormalGameToL>((const void*)pChar);

	return true;
}